charts/ontology/values.yaml

```diff
@@ -51,6 +51,17 @@ probes:
       initialDelaySeconds: 60
       periodSeconds: 10
       failureThreshold: 30 # 5 minutes before restarted
+  startup:
```

Comment:
another chart that doesn't actually need a startupProbe, sorry! This one should be rolled back as well.

Reply:
e732039